sal_interfaces/Dome/Dome_Events.xml

```diff
@@ -1,268 +1,157 @@
-    <Alias>brakeDisengaged</Alias>
-    <Explanation>http://sal.lsst.org</Explanation>
-            <item>
-                <EFDB_Name>brakeDisengagedId</EFDB_Name>
-                <Description>Identify the device whose brake has been disengaged</Description>
-                <IDL_Type>int</IDL_Type>
-                <Units>unitless</Units>
-                <Count>1</Count>
-            </item>
-</SALEvent>
-<SALEvent>
-    <Subsystem>Dome</Subsystem>
-    <Version>3.8.42</Version>
-    <Author></Author>
-    <EFDB_Topic>Dome_logevent_craneStatusChange</EFDB_Topic>
-    <Alias>craneStatusChange</Alias>
-    <Explanation>http://sal.lsst.org</Explanation>
-            <item>
-                <EFDB_Name>craneStatusChangeCode</EFDB_Name>
-                <Description>Return the code of the crane status change</Description>
-                <IDL_Type>int</IDL_Type>
-                <Units>unitless</Units>
-                <Count>1</Count>
-            </item>
-</SALEvent>
-<SALEvent>
-    <Subsystem>Dome</Subsystem>
-    <Version>3.8.42</Version>
-    <Author></Author>
-    <EFDB_Topic>Dome_logevent_rearDoorStatusChange</EFDB_Topic>
-    <Alias>rearDoorStatusChange</Alias>
-    <Explanation>http://sal.lsst.org</Explanation>
-            <item>
-                <EFDB_Name>rearDoorStatusChangeCode</EFDB_Name>
-                <Description>Return the code of the rear door status change</Description>
-                <IDL_Type>int</IDL_Type>
-                <Units>unitless</Units>
-                <Count>1</Count>
-            </item>
-</SALEvent>
-<SALEvent>
-    <Subsystem>Dome</Subsystem>
-    <Version>3.8.42</Version>
-    <Author></Author>
-    <EFDB_Topic>Dome_logevent_sealStatusChange</EFDB_Topic>
-    <Alias>sealStatusChange</Alias>
-    <Explanation>http://sal.lsst.org</Explanation>
-            <item>
-                <EFDB_Name>seatStatusChangeCode</EFDB_Name>
-                <Description>Return the code of the seal status change</Description>
-                <IDL_Type>int</IDL_Type>
-                <Units>unitless</Units>
-                <Count>1</Count>
-            </item>
-</SALEvent>
-<SALEvent>
-    <Subsystem>Dome</Subsystem>
-    <Version>3.8.42</Version>
-    <Author></Author>
-    <EFDB_Topic>Dome_logevent_interlockAlarm</EFDB_Topic>
-    <Alias>interlockAlarm</Alias>
-    <Explanation>http://sal.lsst.org</Explanation>
-            <item>
-                <EFDB_Name>interlockAlarmCode</EFDB_Name>
-                <Description>Return the code of the interlock alarm</Description>
-                <IDL_Type>int</IDL_Type>
-                <Units>unitless</Units>
-                <Count>1</Count>
-            </item>
-</SALEvent>
+    <SALEvent>
+        <Subsystem>Dome</Subsystem>
+        <EFDB_Topic>Dome_logevent_brakesEngaged</EFDB_Topic>
+        <item>
+            <EFDB_Name>brakes</EFDB_Name>
+            <Description>Bitmask of brakes that are engaged.
+            The bits are defined by the Brake enum.</Description>

```

Comment:
Where is this enum defined?

Reply:
It will be defined in the Events file once I know the names of the brakes. If they only have numbers then are they numbered from 0 or 1?

Reply:
We haven't discussed this yet. My assumption was that the brakes are all engaged or none are. So let's clarify this and then make an update to this event.